Examples/UIExplorer/js/ListViewExample.js

```diff
@@ -62,9 +62,9 @@ var ListViewSimpleExample = React.createClass({
         noSpacer={true}
         noScroll={true}>
         <ListView
+          removeClippedSubviews={true}
           dataSource={this.state.dataSource}
           renderRow={this._renderRow}
-          renderScrollComponent={props => <RecyclerViewBackedScrollView {...props} />}
```

Comment:
Got a PR already to get rid of it if you want to shipit :) https://github.com/facebook/react-native/pull/11999